index.rst

```diff
@@ -52,10 +52,10 @@
 News
 ----
 
-- **Release!** Version 0.20.0 2023-02-28
-- **Release!** Version 0.19.3 2022-06-12
-- `CZI announces second round of funding for scikit-image
-  <https://chanzuckerberg.com/newsroom/awards-3-8-million-open-source-software-projects-essential-science/>`__ 2020-03-27
+- **Release!** `Version 0.21.0 <https://scikit-image.org/docs/stable/release_notes/release_0.21.html>`_ 2023-06-02
+- **Release!** `Version 0.20.0 <https://scikit-image.org/docs/0.20.x/release_notes.html>`_ 2023-02-28
+- `scikit-image selected for third round of funding by CZI
```

Comment:
Maybe we can be a bit more explicit here: "Funding received for X as part of CZI's 5-th EOSS cycle"

Reply:
Ok. What about:
"As part of CZI's 5-th EOSS grant cycle, scikit-image received funding for creating a typed, discoverable, and extensible API!"
?

Reply:
That sounds great!